app/code/Magento/Sales/Block/Adminhtml/Order/View/Tab/History.php

```diff
@@ -203,20 +147,6 @@ public function getItemComment(array $item)
             ? $this->adminHelper->escapeHtmlWithLinks($item['comment'], $allowedTags) : '';
     }
 
-    /**
-     * Map history items as array
-     *
-     * @param string $label
-     * @param bool $notified
-     * @param \DateTimeInterface $created
-     * @param string $comment
-     * @return array
-     */
-    protected function _prepareHistoryItem($label, $notified, $created, $comment = '')
```

Comment:
We can't remove protected methods according to backward compatibility policy, please, revert and mark method as deprecated. More information about BIC you can find [here](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/)